### PR TITLE
EV-105 Update Docs to reflect COB changes included in SUCCESS-201

### PR DESCRIPTION
**Pull request by Brooks-morrison**
Need to add changes made in  SUCCESS-201  to docs. Will want to include new fields, descriptions, and example.
Options

### CODE REVIEW

File: source/includes/_claims.md [hunk 1]
```diff
@@ -4731,6 +4731,166 @@ try client.claims(params: data)
...
+            {
+                "charge_amount": "570", 
+                "diagnosis_codes": [
+                    "Z930"
+                ], 
+                "is_emergency": false, 
+                "ordering_provider": {
+                    "first_name": "STEVEN", 
+                    "last_name": "DOE", 
+                    "npi": "1912301953"
```
Comment (bstmartin) [inline source/includes/_claims.md]: Can we update so docs will not have the PD NPI?  Can just use 1234567890

File: source/includes/_claims.md [hunk 2]
```diff
@@ -4731,6 +4731,166 @@ try client.claims(params: data)
...
+                "PO BOX 301404"
+            ], 
+            "city": "CHARLESTON", 
+            "state": "SC", 
+            "zipcode": "29407"
+        }, 
+        "id": "12345", 
+        "organization_name": "CHARLESTON MEDICARE PLAN"
+    }, 
+    "receiver": {
```
Comment (bstmartin) [inline source/includes/_claims.md]: Can we remove receiver, payer, and submitter as these are input by PD after the submission?

File: source/includes/_claims.md [hunk 3]
```diff
@@ -4731,6 +4731,166 @@ try client.claims(params: data)
...
+                "first_name": "EMMIE", 
+                "last_name": "DOE", 
+                "member_id": "519108745", 
+                "payer_responsibility": "tertiary", 
+                "relationship": "self", 
+                "release_of_information_code": "signed_statement"
+            }
+        }
+    ], 
+    "correlation_id": "C0200000000001150965",
```
Comment (bstmartin) [inline source/includes/_claims.md]: Can we remove correlation_id as it is assigned after submission?

File: source/includes/_claims.md [hunk 4]
```diff
@@ -4986,9 +5146,10 @@ The /claims/ response contains an activity and thus returns the same object as t
 
 <a name="claims-coordination-of-benefits-object"></a>
 ###Coordination of Benefits object:
+The coordination of benefits object can be either a list or a dict. If two payers are included in the claim (one payer in COB), COB will be a dict. If three or more payers are included (two+ payers in COB), COB will be a list.
```
Comment (bstmartin) [inline source/includes/_claims.md]: Do we have a maximum value defined for this?  If so, can we list what the max allowable payers would be?

File: source/includes/_claims.md (hunk 1 above)
Comment (Brooks-morrison) [inline source/includes/_claims.md]: Done

File: source/includes/_claims.md (hunk 3 above)
Comment (Brooks-morrison) [inline source/includes/_claims.md]: Done

File: source/includes/_claims.md (hunk 2 above)
Comment (Brooks-morrison) [inline source/includes/_claims.md]: I'll take these out of the current  (2 payer) COB object as well

Comment (Brooks-morrison) [inline source/includes/_claims.md]: Removed from each library

File: source/includes/_claims.md (hunk 4 above)
Comment (Brooks-morrison) [inline source/includes/_claims.md]: Can support up to 11 payers. Will add.

**Comment by denisekgosnell** [comment]
@Brooks-morrison and/or @bstmartin please lmk when this is 👍

**Pull request merged**